data/human/culture conversations.txt

```diff
@@ -80,11 +80,39 @@ outfit "Mug"
 
 
 
+mission "New Holand Foibles"
+	minor
+	source "New Holland"
+	to offer
+		random < 1
+	on offer
+		conversation
+			`The smell of burning metal fills the air as you walk down the uncrowded streets of the factory town near the spaceport, keeping away from the arterial supply route directly feeding the port and its hangars.`
+			`	Despite the distance and absent traffic though it is far from quiet. Several factory buildings make discernible thrumming noises, and grinders echo amidst the silence as you investigate. Turning a corner, you come across an alley running along a lengthy chain link fence line which obstructs your path to anything interesting, beyond which you can see a throng of men cutting away at a rather well kept spaceship. Parts regularly fall to the floor below, apparently heedless of those working below.`
+			`	Even as you watch you see a hull plate swinging dangerously above a workers head.`
+			choice
+				`	(Call out a warning.)`
+				`	(Leave them be.)`
+					goto leave
+			`	You call out a warning and most of the men pause to look at you, the hull plate swinging gently as the holder of the offending grinder squints in your direction. One of the older men, who you now realise was at least somewhat supervising the activity, steps forward and flicks his hands at you in a shooing motion, "Oi you, piss off. We've got our safety supervisor, we don't need your nose in our business."`
+			choice
+				`	(Challenge the claim.)`
+				`	(Leave them be.)`
+					go to leave
+			`	You point out that what you were seeing did not look at all safe, supervised or not, and the man glances around briefly before huffily making his way over to stand opposite you behind the fence. "Look stranger," he says, "work here is hard to find. For those working here, we don't have the time to spare. But the pay is good and we can at least feed our kids. Sure some people get a few scrapes and bruises but no one has died on my watch, and Rik only lost his smallest finger. If you feel the need to watch then fine, stand around, but leave us be ok. There ain't no one around here that'll care what you have to say anyway."`
```

Comment:
```suggestion
			`	You point out that what you were seeing did not look at all safe, supervised or not, and the man glances around briefly before huffily making his way over to stand opposite you behind the fence. "Look stranger," he says, "work here is hard to find. For those working here, we don't have the time to spare. But the pay is good and we can at least feed our kids. Sure, some people get a few scrapes and bruises but no one has died on my watch, and Rik only lost his smallest finger. If you feel the need to watch then fine, stand around, but leave us be. There ain't no one around here that'll care what you have to say anyway."`
```

Reply:
```suggestion
`	You point out that what you were seeing did not look at all safe, supervised or not, and the man glances around briefly before huffily making his way over to stand opposite you behind the fence. "Look stranger," he says, "work here is hard to find. For those working here, we don't have the time to spare. But the pay is good and we can at least feed our kids. Sure, some people get a few scrapes and bruises but no one has died on my watch, and Rik only lost his smallest finger. If you feel the need to watch then fine, stand around, but leave us be okay. There ain't no one around here that'll care what you have to say anyway."`
```

Reply:
Would it not be "Leave us be, okay?"

Reply:
```suggestion
			`	You point out that what you were seeing did not look at all safe, supervised or not, and the man glances around briefly before huffily making his way over to stand opposite you behind the fence. "Look stranger," he says, "work here is hard to find. For those working here, we don't have the time to spare. But the pay is good and we can at least feed our kids. Sure, some people get a few scrapes and bruises, but no one has died on my watch, and Rik only lost his smallest finger. If you feel the need to watch then fine, stand around, but leave us be, okay? There ain't no one around here that'll care what you have to say anyway."`
```
Adding the comma reinforces how I think it would be said out loud, but personal preference.